src/RowRenderer.tsx

```diff
@@ -1,40 +1,26 @@
 import React, { memo } from 'react';
 
 import Row from './Row';
-import RowGroup from './RowGroup';
 import { DataGridProps } from './DataGrid';
-import { RowRendererProps, RowData, CalculatedColumn } from './common/types';
-import EventBus from './EventBus';
+import { RowRendererProps } from './common/types';
 
 type SharedDataGridProps<R, SR> = Pick<DataGridProps<R, never, SR>,
-  | 'rowGroupRenderer'
   | 'rowRenderer'
   | 'onRowClick'
-  | 'onRowExpandToggle'
 >;
 
-interface IRowRendererProps<R, SR> extends SharedDataGridProps<R, SR> {
-  viewportColumns: readonly CalculatedColumn<R, SR>[];
-  lastFrozenColumnIndex: number;
-  rowIdx: number;
-  row: R;
-  eventBus: EventBus;
-  isRowSelected: boolean;
-}
+type IRowRendererProps<R, SR> = RowRendererProps<R, SR> & SharedDataGridProps<R, SR>;
 
 function RowRenderer<R, SR>({
   viewportColumns,
   lastFrozenColumnIndex,
   eventBus,
   rowIdx,
   row,
-  rowGroupRenderer,
   rowRenderer,
   isRowSelected,
-  onRowClick,
-  onRowExpandToggle
-}: IRowRendererProps<R, SR>) {
-  const { __metaData } = row as RowData;
+  onRowClick
+}: RowRendererProps<R, SR> & SharedDataGridProps<R, SR>) {
```

Comment:
I wonder if we should use memo on the `Row` component and delete `RowRenderer`. Thoughts?

Reply:
We still want to have the ability to customize the rowrenderer, right? I believe that's the reason why we kept this layer.

Reply:
Let's just delete this module, memoize `Row`, and use the Row/rowRenderer directly.
Custom row renderers can memo themselves.